.github/workflows/semver-checks-pr-label.yml

```diff
@@ -0,0 +1,87 @@
+on:   # yamllint disable-line rule:truthy
+  workflow_run:
+    workflows: [Check semver breaks]
+    types: [completed]
+
+name: Check semver breaks - Label and Comment PR
+
+jobs:
+  Download:
+    name: Download, Unzip and Add Labels/Comments
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+    # only run if CI passes on the "Check semver breaks" workflow
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - name: "Download artifact"
+        uses: actions/github-script@v7
+        with:
+          script: |
+            // get all artifacts from the workflow run
+            let allArtifacts = await github.rest.actions.listWorkflowRunArtifacts({
+               owner: context.repo.owner,
+               repo: context.repo.repo,
+               run_id: context.payload.workflow_run.id,
+            });
+
+            // find the artifact that starts with 'semver-break'
+            let matchArtifact = allArtifacts.data.artifacts.filter((artifact) => {
+              return artifact.name.startsWith('semver-break');
+            })[0];
+
+            // if no artifact found, exit
+            if (!matchArtifact) {
+              console.log('No semver-break artifact found');
+              process.exit(0);
+            }
```

Comment:
I think that job failing will show as whole CI failed. Or at least, it should.

Reply:
This is safe since in https://github.com/rust-bitcoin/rust-bitcoin/blob/cb6ee619759f1cccc4bb032a2a12edfee8430640/.github/workflows/semver-checks-pr-label.yml#L16-L17 we only run this whole job if the original semver-check is successful.
And it being successful generates either a `semver-break` file or not.